docs/samples/plugin-encryption/style.css

```
@@ -0,0 +1,356 @@
+  margin: 0;
+}
+fieldset legend {
+  font-weight: bold;
+  font-size: 1.1rem;
+}
+
+button, input[type="button"] {
+  position: relative;
+  display: inline-block;
+  min-width: 4.5rem;
+  font-family: CiscoSansTT Regular, Helvetica Neue, Helvetica, sans-serif;
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Helvetica

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)